square.py

```diff
@@ -0,0 +1,61 @@
+# We have a set of points described by (x, y) coordinates in the two dimensional space.
+# Create a function, that calculates coordinates of the smallest possible square (coord and size) that fits all of them.
+# Two sides of the square are expected to be parallel to the x-axis
+# Program prints results on console and draws points and square in a separate window
+
+import json
+import argparse
+import matplotlib.pyplot as plt
+
+# Uploading points from JSON file
+parser = argparse.ArgumentParser()
+parser.add_argument("-f", "--filename", required=True, type=str)
+args = parser.parse_args()
+with open(args.filename) as json_data:
+    points = json.load(json_data)
+    print("Points from file : ", points)
+
```

Comment:
Same here: you assume, that file will be in good standing and contains right format. Mess it up: create a file that contains (a) invalid json (b) valid json, but not list of the points, but something else